library/std/src/os/windows/process.rs

```diff
@@ -5,13 +5,16 @@
 #![stable(feature = "process_extensions", since = "1.2.0")]
 
 use crate::ffi::OsStr;
+use crate::io;
 use crate::os::windows::io::{
     AsHandle, AsRawHandle, BorrowedHandle, FromRawHandle, IntoRawHandle, OwnedHandle, RawHandle,
 };
 use crate::process;
 use crate::sealed::Sealed;
 use crate::sys;
 use crate::sys_common::{AsInner, AsInnerMut, FromInner, IntoInner};
+#[unstable(feature = "windows_process_extensions_raw_attribute", issue = "114854")]
+pub use sys::process::{ProcThreadAttributeList, ProcThreadAttributeListBuilder};
```

Comment:
I don't exactly know the right way to export this here. @Dylan-DPC can you guide me on this?

Reply:
As with the imports above, you need to start the import with `crate::`. But see my review comment.